app/models/publication.rb

```diff
@@ -67,9 +67,10 @@ def self.find_by_doi(doi)
   end
 
   def self.find_by_pmid_pub_id(pmid)
-    Publication.includes(:publication_identifiers)
-               .where("publication_identifiers.identifier_type": 'pmid', "publication_identifiers.identifier_value": pmid)
-               .find_by('wos_uid IS NOT null OR sciencewire_id IS NOT NULL OR pmid IS NOT null')
+    find_by_pmid(pmid) ||
```

Comment:
why not just quickly look to see if we can find the publication via the `.pmid` column in the publication model itself before digging into PublicationIdentifier association?